app/lib/rocketchat.js

```diff
@@ -421,6 +421,10 @@ const RocketChat = {
 		return user;
 	},
 	logout,
+	logoutOtherLocations() {
+		const { id: userId } = reduxStore.getState().login.user;
+		return this.methodCallWrapper('logoutOtherClients', { id: userId });
```

Comment:
We're trying to move all method calls to REST endpoints.
Can you change here to use the proper REST API?
https://docs.rocket.chat/api/rest-api/methods/users/removeothertokens
Thanks.

Reply:
Done